featured-single-post.php

```diff
@@ -1,45 +1,72 @@
 <?php
 /**
  * Template Name: Featured Post
- * Template Post Type:
+ * Template Post Type: Post
  */
 disallow_direct_load( 'featured-single-post.php' );
 get_header( 'featured' ); the_post();
 
+$title = $post->post_title;
+$cats = wp_get_post_categories($post->ID);
+$category_title = get_cat_name($cats[0]);
 $subtitle = get_post_meta( $post->ID, 'subtitle', TRUE );
+$deck = get_post_meta( $post->ID, 'deck', TRUE );
+$source = get_post_meta( $post->ID, 'source', TRUE);
+$img_attach = get_img_html($post->ID, 'story_feature', Array('return_id' => True));
+
+if($img_attach['attachment_id'] != '') {
+	$attachment = get_post($img_attach['attachment_id']);
+}
+
+$video_url = get_video_url($post->ID);
 
 ?>
-<div class="featured-story-header">
-	<div class="row-fluid">
-		<div class="span5">
-			<?php the_post_thumbnail( null, array( 'class' => 'img-responsive' ) ); ?>
-		</div>
-		<div class="span7">
-			<div id="page-title" class="featured-title">
-				<h1><?php echo the_title(); ?></h1>
-				<?php if ( $subtitle ) : ?>
-				<p id="subtitle"><em><?php echo $subtitle; ?></em></p>
-				<?php endif; ?>
-				<div id="deck" class="featured-deck">
-					<p class="lead"><?php the_excerpt(); ?></p>
+
+<div id="feature-story">
+	<div class="container">
+		<div class="row">
+			<div class="span12">
+				<div class="feature-headlines">
+					<h2 class="category-title"><?php echo $category_title; ?></h2>
```

Comment:
Semantically, the h2 -> h1 -> h3 ordering here doesn't make sense, and we should avoid using headings for tagline/subtitle text (https://www.w3.org/TR/2016/REC-html51-20161101/common-idioms-without-dedicated-elements.html#subheadings-subtitles-alternative-titles-and-taglines).

Maybe we could make the `h2.category-title` a `<strong>` tag instead of a heading to help maintain emphasis, and update h3.subtitle to be a `<p>`?